src/model/resources/thermal/thermal.jl

```diff
@@ -30,8 +30,16 @@ function thermal!(EP::Model, inputs::Dict, setup::Dict)
 
 	# Capacity Reserves Margin policy
 	if setup["CapacityReserveMargin"] > 0
-		@expression(EP, eCapResMarBalanceThermal[res=1:inputs["NCapacityReserveMargin"], t=1:T], sum(dfGen[y,Symbol("CapRes_$res")] * EP[:eTotalCap][y] for y in THERM_ALL))
+        reserves = inputs["NCapacityReserveMargin"]
```

Comment:
Suggest calling this something other than "reserves". "Reserves" usually means operating reserves, and not the capacity reserve margin. It is tempting to think of the capacity reserve margin as another type of operating reserve, but in reality it is just accounting for the fact that GenX only models one weather year.